tests/functional_tests/models/bailing/test_bailing_moe2_conversion.py

```
@@ -0,0 +1,389 @@
+        config.torch_dtype = torch.bfloat16  # Explicitly set the torch_dtype in config
+
+        # Create model with random weights and convert to bfloat16
+        # The config loading above should have registered the model class via trust_remote_code
+        try:
+            from transformers import AutoModelForCausalLM
+
+            # Create the model from config - the custom class should be registered from loading config above
+            model = AutoModelForCausalLM.from_config(
+                config, trust_remote_code=True
+            )
+        except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)